python/mlx/nn/layers/activations.py

```diff
@@ -249,11 +309,41 @@ def __call__(self, x):
 
 @_make_activation_module(relu6)
 class ReLU6(Module):
+    r"""Applies the Rectified Linear Unit 6.
+        Applies :math:`\min(\max(x, 0), 6)` element wise.
+
+    See :func:`relu6`, for the functional equivalent.
+    """
+    pass
+
+
+@_make_activation_module(softmax)
+class Softmax(Module):
+    r"""Applies the Softmax function.
+        Applies :math:`\frac{e^{x_i}}{\sum_j e^{x_j}}` element wise.
```

Comment:
Remove the second line here, we can just point to the function below. Could you make that change for all of the Module docstrings you added?